apps/bc-wallet-showcase-creator/components/nav-projects.tsx

```diff
@@ -9,30 +9,33 @@ import {
 } from '@/components/ui/sidebar'
 import { Link, usePathname, useRouter } from '@/i18n/routing'
 import { cn } from '@/lib/utils'
+import { getTenantId, useTenant } from '@/providers/tenant-provider'
 import { CreditCard, Home, Map } from 'lucide-react'
 import { useTranslations } from 'next-intl'
 
-const projects = [
-  {
-    name: 'home_label',
-    url: '/',
-    icon: Home,
-  },
-  {
-    name: 'showcases_label',
-    url: '/showcases',
-    icon: Map,
-  },
-  {
-    name: 'credential_library_label',
-    url: '/credentials',
-    icon: CreditCard,
-  },
-]
-
 export function NavProjects() {
   const t = useTranslations('sidebar')
   const pathname = usePathname()
+  const { tenantId } = useTenant();
+
+  const projects = [
+    {
+      name: 'home_label',
+      url: `/${tenantId}`,
```

Comment:
my idea at start was to add on the middleware level the tenant id to avoid to declated it on all screens